internal/test/testdata/compile/bad_cel_expr.yaml

```diff
@@ -3,9 +3,9 @@
 wantErrors:
   - file: principal_policies/donald_duck_20210210.yaml
     error: |-
-      failed to compile `resource.attr.dev_record ^^ true` [Syntax error: token recognition error at: '^', Syntax error: token recognition error at: '^', Syntax error: extraneous input 'true' expecting <EOF>]
+      invalid expression `resource.attr.dev_record ^^ true`
```

Comment:
If we have the expression in the description, do we need it in the error message also? It would be better to keep the error to a couple of words to match the others. There is not much room in the Hub UI where we display this, and the expression could be quite long.